client/src/components/layouts/certification.tsx

```diff
@@ -1,47 +1,48 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
+import React, { Fragment, Component } from 'react';
```

Comment:
```suggestion
import React, { Component } from 'react';
```
In combination with Oliver's suggestion, we'll need to remove the unused import.